roles/core/dns_server/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+# implement more default values which will come from inventory
+j2_current_iceberg_network: ice-0
```

Comment:
j2 variables should never be redefined, as they are hearth of the stack. Only manual precedence is expected.
Please note also that networks and icebergs start numbering at 1, not 0 in BlueBanquise :-)

Reply:
ok, then I will move them in the converge.yml as pre_tasks, ok?

Reply:
Yes. May I ask why you need to force j2_current_iceberg_network: ice-0 ?
Is it because you are using chroot environment ?

Reply:
We don't load the example inventories with Molecule.
We either have to explicitly define mandatory variables in an inventory internal to the Molecule scenario (converge.yml), or rely on defaults values. Since usage of default values is not an option here, we have to define the mandatory variables in converge.yml.
We can define different scenarios with different values to do multiple tests. (eg. one scenario to test a role behaviour with single iceberg and another scenario to test with multi iceberg)